### ISSUE TITLE
Edit two Questions: 'ua_ruhr_v1.1.0-beta.1/technical-classification/data-volume/rate'  and  'ua_ruhr_v1.1.0-beta.1/technical-classification/data-volume/volume'

### ISSUE THREAD

**Issue by smohex** [FDM-UARuhr/rdmo-catalog-uaruhr#57]
**Describe the request**  
question should not be optional, question about size is relevant in any case

**Propose what exactly should be changed**  
new help text (see below)

**Original setting or content**   
*help text ger*  
"Optional. Dies ist nur relevant, wenn das Wachstum die TB-Größenordnung erreicht."
`technical-classification/data-volume/volume`  

**Proposed setting or content**   
1. `technical-classification/data-volume/rate`  

* remove help text 

2. `technical-classification/data-volume/volume`  

*help text ger* 
"Wenn große Datenmengen anfallen sollten finanzielle Mittel zur Bereitstellung der Infrastruktur berücksichtigt werden (siehe auch Abschnitt „Förderung“)"
*help text en*  
"If large amounts of data are involved, financial resources for the provision of the infrastructure should be considered (see also section "Funding")"

**From the following list delete everything that is NOT part of the content of your proposal. Once a subject of change is approved it will be marked accordingly in the list.**  

- [x] help text (en and ger)

**Comment by jwindeck** [comment]
removing "optional" is fine, but a better place for the proposed help text might be: ``technical-classification/data-volume/volume``

**Comment by smohex** [comment]
UDE/TUDO: we agree with Jürgen's proposal

**Comment by smohex** [comment]
Help text completely removed

**Comment by jmfrenzel** [comment]
The change to the texts was applied to the wrong question /technical-classification/data-volume/volume    https://github.com/FDM-UARuhr/rdmo-catalog-uaruhr/blob/74a0a0ded748961ad28c896be25b5ae4efa97938/tools/all_questions.xml#L1077

But the change concerns /technical-classification/data-volume/rate

**Comment by ClaraRosalie** [comment]
Changes should be implemented in upstream as well

**Comment by ClaraRosalie** [comment]
PR in upstream #112
